src/app/onboarding/components/onboarding/onboarding.component.stories.ts

```diff
@@ -230,3 +261,10 @@ Glam.args = {
   hasInvitations: false,
   isGlam: true,
 };
+
+export const GlamShareInvite: Story = StoryTemplate.bind({});
+GlamShareInvite.args = {
+  accountName: 'Test User',
+  isGlam: true,
+  hasShareToken: true,
```

Comment:
```suggestion
  hasInvitations: false,
  shareToken: true,
```
This argument name isn't correct and we also probably want to be explicit about turning off invitations in this flow. We also probably want to be explicit about disabling the share token in all other stories.